src/components/views/messages/MImageBody.js

```diff
@@ -40,15 +41,37 @@ module.exports = React.createClass({
         onWidgetLoad: PropTypes.func.isRequired,
     },
 
+    contextTypes: {
+        matrixClient: PropTypes.instanceOf(MatrixClient),
+    },
+
     getInitialState: function() {
         return {
             decryptedUrl: null,
             decryptedThumbnailUrl: null,
             decryptedBlob: null,
             error: null,
+            imgError: null,
```

Comment:
why `null` rather than `false`?

(a comment describing what it means wouldn't hurt)

Reply:
No reason, probably because the others are `null` and it doesn't actually have implications on the functionality.

`false` is probably clearer.